src/typing-utils/operations.d.ts

```diff
@@ -10,6 +10,9 @@ export interface CreateOperationConfig {
   returnEntity: boolean;
   delay: number;
   responseModel: ResponseModel;
+  extensions?: {
+    withDefaultValues: Record<string, any>
+  }
```

Comment:
you can have extensions in general as well, for example in the case you want to add some property all the time.